rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -1165,6 +1162,42 @@ void should_KeepFiltersInTheLinkOfTheResponse_When_GettingTasks() {
                   + "&sort-by=POR_VALUE&order=DESCENDING");
     }
 
+    @Test
+    void should_ReturnTasksWithOwnerNull_When_GettingTasks() {
+      String url = restHelper.toUrl(RestEndpoints.URL_TASKS) + "?owner-is-null";
+      HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("admin"));
+
+      ResponseEntity<TaskSummaryPagedRepresentationModel> response =
+          TEMPLATE.exchange(url, HttpMethod.GET, auth, TASK_SUMMARY_PAGE_MODEL_TYPE);
+
+      assertThat(response.getBody()).isNotNull();
+      assertThat((response.getBody()).getLink(IanaLinkRelations.SELF)).isNotNull();
+      assertThat((response.getBody()).getContent()).hasSize(65);
+    }
+
+    @Test
+    void should_ReturnTasksWithOwnerNullOrUser11_When_GettingTasks() {
+      String url = restHelper.toUrl(RestEndpoints.URL_TASKS) + "?owner-is-null&owner=user-1-1";
+      HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("admin"));
+
+      ResponseEntity<TaskSummaryPagedRepresentationModel> response =
+          TEMPLATE.exchange(url, HttpMethod.GET, auth, TASK_SUMMARY_PAGE_MODEL_TYPE);
+
+      assertThat(response.getBody()).isNotNull();
+      assertThat((response.getBody()).getLink(IanaLinkRelations.SELF)).isNotNull();
+      assertThat((response.getBody()).getContent()).hasSize(75);
+    }
+
+    @Test
+    void should_ThrowException_When_OwnerIsNullParamNotStrict() throws Exception {
+      String url = restHelper.toUrl(RestEndpoints.URL_TASKS) + "?owner-is-null=";
+      HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("admin"));
+
+      assertThatThrownBy(
```

Comment:
We should discuss if it should throw an Exception or return an http error response